gateways/checkoutcom/request_builder.go

```diff
@@ -7,6 +7,12 @@ import (
 	"github.com/checkout/checkout-sdk-go/payments"
 )
 
+// Cof specifies the transaction type under the Credential-on-File framework
+const (
+	cofCIT = "CIT"	// Customer Initiated Transaction
+	cofMIT = "MIT"	// Merchant Initiated Transaction
```

Comment:
It doesn't look like these are used for anything, should we add a const for "Recurring" instead?

Reply:
Good call, done